target_position: [0.0, 0.0]

target_color: 'red'
estimate_color: 'green'
agent_color: 'blue'
